nautobot/extras/tests/test_models.py

```diff
@@ -309,7 +310,7 @@ def test_multiple_tags_return_distinct_objects(self):
         device.tags.add(self.tag2)
 
         annotated_queryset = Device.objects.filter(name=device.name).annotate_config_context_data()
-        self.assertEqual(ConfigContext.objects.get_for_object(device).count(), 1)
+        self.assertEqual(ConfigContext.objects.get_for_object(device).count(), 2)  # including one from setUp()
```

Comment:
Non-blocking: can we use `self.assertQuerysetEqualAndNotEmpty` here?

Reply:
I'd be more tempted to remove this honestly. It's not a filter test so there's not really a different queryset to compare against that I can see.